### PR TITLE
E2e create new wallet + import wallet

### PR DESCRIPTION
**Pull request by oryhivskiy**

### CODE REVIEW

File: ios/TempleWallet.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -520,6 +520,8 @@
 				CURRENT_PROJECT_VERSION = 41;
 				DEVELOPMENT_TEAM = 9DBX2BVVMN;
 				ENABLE_BITCODE = NO;
+				EXCLUDED_ARCHS = arm64;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```
Comment (unixvb) [inline ios/TempleWallet.xcodeproj/project.pbxproj:524]: revert all changes in this file

File: src/components/mnemonic/mnemonic-create/mnemonic-create.tsx [hunk 2]
```diff
@@ -43,13 +44,22 @@ export const MnemonicCreate: FC<MnemonicProps> = ({ value, isError, onChangeText
...
+          testID={CreateNewWalletSelectors.GenNewSeedButton}
+        />
         {isString(value) ? (
           <>
             <Divider size={formatSize(8)} />
-            <ButtonSmallSecondary title="COPY" onPress={() => copyStringToClipboard(value)} />
+            <ButtonSmallSecondary
+              title="COPY"
+              onPress={() => copyStringToClipboard(value)}
+              testID={CreateNewWalletSelectors.CopyButton}
```
Comment (unixvb) [inline src/components/mnemonic/mnemonic-create/mnemonic-create.tsx:61]: You are using `CreateNewWalletSelectors` inside `MnemonicCreate` component. 
This's wrong.  Selectors should only be used in component where they are declared

**Review by unixvb — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]

**Comment by IrynaKhyzhynska** [comment]
ok

**Review by IrynaKhyzhynska — APPROVED** [review]
ok

**Review by IrynaKhyzhynska — APPROVED** [review]

**Pull request merged**